src/RadixDlt.NetworkGateway.GatewayApi/gateway-api-schema.yaml

```diff
@@ -2848,6 +2861,24 @@ components:
               type: array
               items:
                 $ref: "#/components/schemas/StreamTransactionsRequestEventFilterItem"
+            accounts_with_manifest_owner_method_calls:
+              type: array
+              items:
+                $ref: "#/components/schemas/Address"
+            accounts_without_manifest_owner_method_calls:
+              type: array
+              items:
+                $ref: "#/components/schemas/Address"
+            transaction_type_filter:
```

Comment:
Ditto - possibly `transaction_class_filter` ?

Reply:
same as above I went for `manifest_class_filter`